patches/minecraft/net/minecraft/item/ItemStack.java.patch

```diff
@@ -118,7 +118,7 @@
  
     public boolean func_77984_f() {
 -      if (!this.field_190928_g && this.func_77973_b().func_77612_l() > 0) {
-+      if (!this.field_190928_g && this.func_77973_b().getMaxDamage(this) > 0) {
++      if (!this.field_190928_g && this.func_77973_b().func_77645_m()) {
```

Comment:
I'm not sure about this. The call to getMaxDamage is itemstack-aware, but it's being patched with a call that is **not** stack-aware, so it seems counter-productive?

Reply:
Did I mess up the Backport ? Ahh, back then we did not have the isDamageable I think.

Reply:
You added isDamageable(ItemStack) in the same PR, but didn't add that stack-sensitive method here, so the change is just kindof standing on one foot.

Reply:
@mcenderdragon Any update on these comments?

Reply:
right now I dont have the energy to port/change this, It will hopefully get better at the end of april. then again it is in 1.16 so it is not as important if 1.15 is missing this.